### PR TITLE
Prepare for v0.1.0

### PR DESCRIPTION
**Pull request by ColinSullivan1**
Prepare for v0.1.0 with goreleaser:
* Add goreleaser to Travis
* Fix coverage output directory
* Add travis_wait temporary log files to .gitignore
* Remove cross compile script (no longer needed)
* Fix readme Build Status references
* Update docker images to build v0.1.0 release

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -2,10 +2,11 @@
 project_name: prometheus-nats-exporter
 
 release:
+  prerelease: true
```
[the remark this answers is not in the document]
Comment (ColinSullivan1) [reply .goreleaser.yml]: Good point, let me look.

[the remark this answers is not in the document]
Comment (ColinSullivan1) [reply .goreleaser.yml]: I suspect that's only for command line releases, but will test.

**Comment by ColinSullivan1** [comment]
@kozlovic , this is interesting.  When set to draft, we get two releases.  One from the GH user, and another from NATS-CI.  So I guess the answer is, we pick which user.  ;)

**Comment by kozlovic** [comment]
@ColinSullivan1 It seems that it builds one per Golang version. You should do it for a single one.

PS: unrelated: could you change the avatar for that user and use the NATS logo like the other nats ci user?

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,11 @@ language: go
 go:
 - 1.8.5
```
Comment (kozlovic) [inline .travis.yml]: Uer 1.8.x and 1.9.x or specify explicit release that is more recent. For 1.9 we are at 1.9.4 at the moment.

Comment (kozlovic) [reply .travis.yml]: Also, you may want to update your Dockerfile's to reference 1.9.4 (or whatever latest you would want it to be built with).

Comment (ColinSullivan1) [reply .travis.yml]: I'll move them all to 1.9.4, and remove 1.8.5.   Although, there is 1.10...

**Comment by ColinSullivan1** [comment]
@kozlovic , thank you for noticing that.  Fixed...   Check out @NATS-CI.

**Comment by ColinSullivan1** [comment]
@kozlovic , I've updated to use go 1.9.4 across the board, and retested all docker images.  IMO we should use the same go version in docker as we do in release builds.

**Comment by kozlovic** [comment]
@ColinSullivan1 Yes, that's usually what we do. What may happen is that in master we update to a newer Go version when avail, but when comes time for the release, both builds and docker images are at the same Go version.

**Comment by ColinSullivan1** [comment]
@kozlovic @aricart , I've added release instructions [here](https://github.com/nats-io/prometheus-nats-exporter/blob/update-travis-gor/RELEASING.md).  Good to go?

**Comment by aricart** [comment]
LGTM!

**Comment by kozlovic** [comment]
@ColinSullivan1 Actually, if your Docker files would have something like:
```
git clone --branch <tag name> https://github.com/nats-io/prometheus-nats-exporter.git .
```
you would be able to checkout a specific branch or tag version. This would allow you to test the docker file first (by specifying a branch) and then you would put the tag of the next release. It means that when you push the tag, you would have the proper git commit. Does that makes sense?

**Comment by ColinSullivan1** [comment]
@kozlovic , as discussed over the phone - we can use tags directly, I'll update the RELEASE.md and docker files.

**Comment by ColinSullivan1** [comment]
@kozlovic @aricart , one more look at [RELEASING.md](https://github.com/nats-io/prometheus-nats-exporter/blob/update-travis-gor/RELEASING.md) would be appreciated.  I also prepared the docker images for v0.1.0.  Someday we can automate further...

**Comment by aricart** [comment]
Looks good. I wonder if this document should go into a private repo, where we collect documentation on how we handle releases and release configuration. That way we have one doc to keep updated and we can simply point to it.

**Comment by kozlovic** [comment]
@ColinSullivan1 @aricart I agree with Alberto, and we do have such repo with release instructions for other products.
Otherwise, rest LGTM.

**Comment by ColinSullivan1** [comment]
Thanks guys, I'll move the doc to our internal repo, and merge.  Thanks for the review!

**Pull request merged**